tools/accuracy_checker/requirements.in

```diff
@@ -8,9 +8,6 @@ scikit-image
 # reid
 scikit-learn
 
-# pipelining mode
-yamlloader
```

Comment:
You should also update `ci/requirements-*`.

Reply:
but before I need to finish clean up in AC, right?

Reply:
What do you mean?

Reply:
PR is in `progress,` possibly I remove something else, when I finish work on this Pr, then I update requirements

Reply:
Sure, take your time. :-)

Reply:
done!